recipes/higra/conda_build_config.yaml

```diff
@@ -0,0 +1,21 @@
+c_stdlib:
+  - sysroot                    # [linux]
+  - macosx_deployment_target   # [osx]
+  - vs                         # [win]
+
+c_stdlib_version:              # [unix]
+  - 2.17                       # [linux]
+  - 11.0                       # [osx]
```

Comment:
Do you really need the last line?

Reply:
I'm not really sure. I'll try removing it.